Vagrantfile

```diff
@@ -48,6 +48,7 @@ vvv_config['sites'].each do |site, args|
   defaults['branch'] = 'master'
   defaults['skip_provisioning'] = false
   defaults['allow_customfile'] = false
+  defaults['upstream'] = 'php'
```

Comment:
How about `nginx_upstream` or `phpfpm_upstream`? I doubt we'll ever use "upstream" elsewhere, but it may make it more clear.